prefect_sqlalchemy/credentials.py

```diff
@@ -156,15 +156,21 @@ def block_initialization(self):
             query=self.query,
         )
         if not self.url:
-            required_url_keys = ("drivername", "username", "database")
+            required_url_keys = ("drivername", "database")
             if not all(url_params[key] for key in required_url_keys):
-                required_url_keys = ("driver", "username", "database")
+                required_url_keys = ("driver", "database")
```

Comment:
Is this duplicate assignment of `required_url_keys` necessary?

Reply:
Yeah, class kwarg is driver, however, the sqlalchemy url requires it to be "drivername", but when it errors, it says drivername is missing, but that's not really the class kwarg; driver is